### ISSUE TITLE
[5.0.0-rc.2] another out of memory

### ISSUE THREAD

**Issue by pkra** [zorkow/speech-rule-engine#876]
Here's one I found using rc2

```
\begin{equation*}
a\mathrel{\leftrightsquigarrow _{\mathrm{}}}_{\mathrm{p}}b\Longleftrightarrow a\mathrel{\rightsquigarrow _{\mathrm{p}}}b\mathrel{\rightsquigarrow _{\mathrm{p}}}a.
\end{equation*}
```

**Comment by pkra** [comment]
Here's another one 

```
\begin{align*}
 \| \partial _{} S \|_{k-1,\flat } &\leq | \partial _{} S - \partial _{} ( S - \partial _{} Q ) |_{k-1} + |S - \partial _{} Q|_{k} = |S - \partial _{} Q|_{k} \leq \| S \|_{k,\flat } + {\epsilon }. 
\end{align*}
```

**Comment by zorkow** [comment]
Thanks for reporting these. Here is a minimal version for the first issue:

```XML
  <msub>                              
    <mrow data-mjx-texclass="REL">
      <msub><mo>↭</mo><mrow/></msub>  
    </mrow>
    <mi>p</mi>
  </msub>
```

**Comment by zorkow** [comment]
This is again an overshoot issue when ascending during enrichment to find a suitable element to annotate. The same guard that allowed the subscript to ignore the empty script element also allowed it ignore the upper level `msub`, climbing all the way to the toplevel `math` elements and inserting this as a 
The reason this leads to OOM and is not caught (as in MJ3) is that the `@xmldom/xmldom` package has a less sophisticated hierarchy loop check than the browser DOM implementation. I.e., appending a node to its great-grandchild is possible and leads to a cyclic structure.

Anyway, I have a fix for the two reported expressions, but I found a few more, similar issues:

```LaTeX
| {|_{k}} S |
| {|^{k}} S |
| {|_{k}}^{} S |
| {|_{k}}_{} S |
| {|^{k}}^{} S |
| {|^{k}}_{} S |
```

I am currently abstracting the fix.

**Comment by pkra** [comment]
More examples

- `\mathbf{1}_{\mathrm{p}_{}}`
- `\begin{equation*} I = I \boxtimes I = F(I) = F(M) \boxtimes F(M) \overset{e}{\xrightarrow {}}F(M \boxtimes M) = I.  \end{equation*}`
- `\begin{equation*}  (\mathbb{P}^1\times \mathbb{P}^1)\times \mathbb{H}^{N,P,p}\supset \mathcal{G}\coloneq \mathrm{graph}(\phi ) \overset{\hat{\phi }}{ \xrightarrow {\hspace{1.8cm}}} \mathbb{H}^{N,P,p},  \end{equation*} `
- `\begin{align}  \int _{-\infty }^\infty \bigg |\frac{\partial ^{} }{\partial x} \eta (x, y)\bigg |^2 \psi (x) \,\mathrm{d} x \,&=\, \int _{-\infty }^y \frac{(\Phi (x))^2}{\psi (x)} \,\mathrm{d} x + \int _{y}^\infty \frac{(1-\Phi (x))^2}{\psi (x)} \,\mathrm{d} x \\  &=\, \eta (y,y), \cssId{texmlid44}{\tag{27}} \end{align} `
- `\begin{align*} \begin{array}{c} \vcenter{\img[][58pt][48pt][{Graphic alt text}]{some.svg}} \end{array} =(-q)\begin{array}{c} \vcenter{\img[][58pt][48pt][{Graphic alt text}]{anothre.svg}} \end{array} .  \end{align*}`
- `\{\operatorname {Sub}_{}(G/N)\}_{N}`
- `{\odot _{  }} _\xi `

**Comment by zorkow** [comment]
Two issues remaining. All others are working fine with PR #878.

### Issue 1:

```LaTeX
\begin{align}  \int _{-\infty }^\infty \bigg |\frac{\partial ^{} }{\partial x} \eta (x, y)\bigg |^2 \psi (x) \,\mathrm{d} x \,&=\, \int _{-\infty }^y \frac{(\Phi (x))^2}{\psi (x)} \,\mathrm{d} x + \int _{y}^\infty \frac{(1-\Phi (x))^2}{\psi (x)} \,\mathrm{d} x \\  &=\, \eta (y,y), \cssId{texmlid44}{\tag{27}} \end{align}
```
can be minimized to `\bigg |\frac{\partial ^{} }{\partial x} \bigg |^2`, where we have an interaction between the empty superscript and the embellished right fence.


### Issue 2:

The following is similar to issues in #881
```LaTeX
\{\operatorname {Sub}_{}(G/N)\}_{N}
```

**Comment by pkra** [comment]
I can confirm that with mathjax v4.1.3 + SRE 5.0.0.rc.3 our use cases are fixed.
